src/topp/DatabaseSuitability.cpp

```diff
@@ -220,6 +225,9 @@ class DatabaseSuitability :
       // skip if the top hit is a decoy hit
       if (top_hit.getMetaValue("target_decoy") == "decoy") continue;
 
+      // skip if top hit is out ouf FDR
```

Comment:
The new statements and maybe some old stuff assumes that the hits are sorted. I wonder if we should add a Precondition or make sure that this is actually the case.

Reply:
Yes, this would probably be a good idea.
Is there a convenient way of doing this? Of cause I can just check while calculating the decoy cut-off..

Reply:
I think a hits.sort() before you do anything on the hits should be fine. Not too much overhead.